### ISSUE TITLE
Translate /subjects into more languages

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#4677]
Creating sub-issues for languages where we've had recent contributions. If folks want to translate into other languages, please comment! And I'll add to the list :) 

- [x] English
- [x] #4675 Spanish (@dcapillae ) https://openlibrary.org/subjects.es
- [x] #4676 French (@lephemere ) https://openlibrary.org/subjects.fr
- [x] #4678 German (@bitnapper ) https://openlibrary.org/subjects.de
- [x] #4679 Croatian (@milotype ) https://openlibrary.org/subjects.hr
- [x] #4682 Russian (@contributorpw ) https://openlibrary.org/subjects.ru

NOTE: This is kind of annoying in that ideally we'd want to #2819 , which would give us the ability to store translated titles on the subject itself. But that's far away unfortunately, and in this seems good enough in the interm, and will provide a lot of value to non-English users. We will also be able to copy over the translations from this page once #2819 is complete, so no one's work is wasted.

**Comment by contributorpw** [comment]
Please add Russian. My OL user is oshliaer

**Comment by cdrini** [comment]
Awesome, thank you @contributorpw ! I created a separate issue: #4682 , and you should now have access to edit here: https://openlibrary.org/subjects.ru?m=edit

**Comment by LeadSongDog** [comment]
Q: Is the plan to use romanized spelling or vernacular?

**Comment by cdrini** [comment]
Vernacular :) I don't thing e.g. Russian users would benefit from having the UI be in romanized Russian.

**Comment by cdrini** [comment]
6 languages complete! 🎉
